graphistry/feature_utils.py

```diff
@@ -32,21 +35,22 @@
         MultiLabelBinarizer,
         KBinsDiscretizer,
     )
+
     has_dependancy = True
 
 except ModuleNotFoundError as e:
     logger.debug(
-        f"AI Packages not found, trying running `pip install graphistry[ai]`",
+        "AI Packages not found, trying running `pip install graphistry[ai]`",
         exc_info=True,
     )
     import_exn = e
     has_dependancy = False
     scipy = (Any,)
     torch = Any
     SuperVectorizer = (Any,)
-    #SimilarityEncoder = (Any,)
-    #TargetEncoder = (Any,)
-    #MinHashEncoder = (Any,)
+    # SimilarityEncoder = (Any,)
+    # TargetEncoder = (Any,)
+    # MinHashEncoder = (Any,)
```

Comment:
Should these look more like `TargetEncoder : Any = None` ? Why is `Any` on the right-hand side, instead of just being a Type?

Reply:
If too annoying, do some sort of ` # type: ignore`